### PR TITLE
bump minor version

### PR DESCRIPTION
**Pull request by mmoll**
To be merged after #59

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 3.3.0
+
+* Add dns_notify to dns class and dns::zone define
+* Add acls hash
+* Add controls hash
+* Change default path for zones.conf to /etc/named/zones.conf for RedHat
```
Comment (ekohl) [inline CHANGELOG.md]: the RedHat osfamily?

Comment (mmoll) [inline CHANGELOG.md]: correct, changed

**Comment by ekohl** [comment]
:+1: and I merged #59 so I think this is good to go.

**Comment by domcleal** [comment]
Thanks!

**Pull request merged**